### PR TITLE
fix: add missing SessionIssued event for api flows

### PR DESCRIPTION
**Pull request by jonas-jonas**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got the approval (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](https://github.com/ory/docs).

### CODE REVIEW

File: selfservice/flow/login/hook.go [hunk 1]
```diff
@@ -190,10 +190,15 @@ func (e *HookExecutor) PostLoginHook(
...
-			SSOProvider: provider,
+			SessionID:    s.ID,
+			IdentityID:   i.ID,
+			FlowType:     string(a.Type),
+			RequestedAAL: string(a.RequestedAAL),
+			IsRefresh:    a.Refresh,
+			Method:       a.Active.String(),
+			SSOProvider:  provider,
 		}))
+		trace.SpanFromContext(r.Context()).AddEvent(events.NewSessionIssued(r.Context(), string(s.AuthenticatorAssuranceLevel), s.ID, s.IdentityID))
```
Comment (hperl) [inline selfservice/flow/login/hook.go:201]: Is this the only place? How about https://github.com/ory/kratos/blob/ae22c7cdb58e2b9bd2d147bf890f95fc85b9aef2/selfservice/hook/session_issuer.go#L55?

Should we maybe rather instrument https://github.com/ory/kratos/blob/ae22c7cdb58e2b9bd2d147bf890f95fc85b9aef2/persistence/sql/persister_session.go#L200 ?

**Review by hperl — CHANGES_REQUESTED** [review]
Thanks for getting on this quickly! Not sure if we hit all places yet, though.

File: selfservice/flow/login/hook.go (hunk 1 above)
Comment (jonas-jonas) [reply selfservice/flow/login/hook.go:201]: Added the post registration hook.

**Comment by aeneasr** [comment]
I agree that this is not very well instrumented yet. The instrumentation should probably happen in the persister. We can merge this as is though IMO. Up to you @jonas-jonas . Also possible that we missed a few spots for example in social sign in

**Review by hperl — APPROVED** [review]
LGTM!

**Comment by aeneasr** [comment]
https://github.com/ory/kratos/pull/3351

**Pull request merged**